src/DynamoCoreWpf/TestInfrastructure/AbstractMutator.cs

```diff
@@ -33,9 +34,37 @@ public virtual Type GetNodeType()
             return typeof(NodeModel);
         }
 
+        /// <summary>
+        /// Number of times to repeat a specific mutation
+        /// </summary>
         public virtual int NumberOfLaunches
         {
-            get { return 1000; }
+            get { return 10; }
+        }
+
+
+
+
+        // TODO(lukechurch): Move this to a support class
```

Comment:
Consider making YT tasks for all of the todo's. Curiously, tasks of this sort never seem to get done...